src/script/assets/AssetRemoteData.ts

```diff
@@ -17,55 +17,89 @@
  *
  */
 
-import {getLogger} from 'utils/Logger';
-import {loadUrlBuffer} from 'utils/util';
-import {getAssetUrl, setAssetUrl} from './AssetURLCache';
+import {Logger} from 'logdown';
```

Comment:
Please use `utils/Logger`.

Reply:
It won't compile. Error: TS2307.

Reply:
I mean, please use the `Logger` interface from 'utils/Logger (add it in line 21).